mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+pretty = true
+show_column_numbers = true
+show_error_codes = true
+warn_no_return = true
+warn_redundant_casts = true
+warn_unused_configs = true
+warn_unused_ignores = true
+ignore_missing_imports = true
+exclude = cheroot/test
```

Comment:
Why is this excluded? Linting tests is as important as linting the rest of the code.

Reply:
I was not aware of that. Not everybody likes types tests...

Reply:
@webknjaz are you sure, that we should add stubs for tests? That sounds weird to me

Reply:
Usually, the whole project needs to be tested. But maybe you're right — it's probably more useful to have in-source annotations for tests rather than in separate files.